specification/webpubsub/resource-manager/Microsoft.SignalRService/stable/2021-10-01/examples/WebPubSubHubs_List.json

```diff
@@ -0,0 +1,42 @@
+{
+  "parameters": {
+    "api-version": "2021-10-01",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "myResourceGroup",
+    "resourceName": "myWebPubSubService"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "properties": {
+              "eventHandlers": [
+                {
+                  "properties": {
+                    "urlTemplate": "http://host.com",
+                    "userEventPattern": "*",
+                    "systemEvents": [
+                      "connect",
+                      "connected"
+                    ],
+                    "auth": {
+                      "type": "ManagedIdentity",
+                      "managedIdentity": {
+                        "resource": "abc"
+                      }
+                    }
+                  },
+                  "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/myResourceGroup/providers/Microsoft.SignalRService/WebPubSub/myWebPubSubService/hubs/exampleHub/eventHandlers/exampleEventHandler",
+                  "name": "exampleEventHandler"
+                }
+              ]
+            },
+            "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/myResourceGroup/providers/Microsoft.SignalRService/WebPubSub/myWebPubSubService/hubs/exampleHub",
+            "name": "exampleHub"
```

Comment:
>"name": [](http://example.com/codeflow?start=12&length=7)

GET resource must have 'type' property for a fully qualified resource type.

[https://armwiki.azurewebsites.net/api_contracts/RPCstrongvalidation.html?q=strong#get-resource](https://armwiki.azurewebsites.net/api_contracts/RPCstrongvalidation.html?q=strong#get-resource)